googleapi/googleapi.go

```diff
@@ -245,12 +246,30 @@ func ChunkSize(size int) MediaOption {
 	return chunkSizeOption(size)
 }
 
+type chunkRetryDeadlineOption time.Duration
+
+func (cd chunkRetryDeadlineOption) setOptions(o *MediaOptions) {
+	o.ChunkRetryDeadline = time.Duration(cd)
+}
+
+// ChunkRetryDeadline returns a MediaOption which sets a per-chunk retry
+// deadline. If a single chunk has been attempting to upload for longer than
+// this time and the request fails, it will no longer be retried, and the error
+// will be returned to the caller.
+// This is only applicable for files which are large enough to require
+// a multi-chunk resumable upload.
+// The default value is 32s.
```

Comment:
Has 32s at per chunk been the default for a while now?

Reply:
For 3ish years: https://github.com/googleapis/google-api-go-client/commit/4fac4de465568dff4e9b2a31b4d7a1da4ba2cefb

Reply:
Yep, and there was a default of 16s prior to that apparently.